addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -29,16 +34,18 @@ For each command a separate Thing has to be defined.
 Thing exec:command:apc [command="/usr/local/bin/apcaccess  status", interval=15, timeout=5, autorun=false]
 ```
 
-`command` itself can be enhanced using the well known syntax of the **java.util.Formatter** class.
+`command` itself can be enhanced using the well known syntax of the **java.util.Formatter** class. 
```

Comment:
Here `command` is the right way to write it, other than for the "command" Thing

Please remove the space at the end.